idea/idea-frontend-api/src/org/jetbrains/kotlin/idea/frontend/api/symbols/KtFunctionLikeSymbol.kt

```diff
@@ -13,6 +13,7 @@ import org.jetbrains.kotlin.name.FqName
 
 public abstract class KtFunctionLikeSymbol : KtCallableSymbol(), KtSymbolWithKind {
     public abstract val valueParameters: List<KtValueParameterSymbol>
+    public abstract val hasStableParameterNames: Boolean
```

Comment:
The meaning of this property is not clear by its name. Maybe some better name + KDoc may be used here?

Reply:
I copied the name from: https://github.com/JetBrains/kotlin/blob/92d200e093c693b3c06e53a39e0b0973b84c7ec5/core/descriptors/src/org/jetbrains/kotlin/descriptors/CallableDescriptor.java#L52-L57

I will copy the KDoc as well.

Reply:
I mean that it is not clear what "stable" parameter name means. Could you please explain this?

Reply:
I copied over the KDoc, is it not clear still what "stable" means? I'm open to suggestions on how to make it better. `hasParameterNamesThatCanBeUsedForCalls` is very specific. I think using "stable" + KDoc to explain is a good balance.

Reply:
This KDoc is totally fine, I just misread it the first time :)